src/Components/addItem.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { fb } from '../lib/firebase';
+import { FirestoreProvider } from 'react-firestore';
+
+class Item extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      itemname: '',
+    };
+  }
+  updateInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+  addItem = e => {
+    e.preventDefault();
+    const db = fb.firestore();
+    db.settings({
+      timestampsInSnapshots: true,
```

Comment:
When executing this function, I found an error in the console. I think you could remove this to prevent the error:
![](https://cdn.zappy.app/f0ec40b28e945e573c25a2ff3893e4d4.png)